examples/prisma-example/apps/web/src/pages/_app/PageConfig/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, Fragment, ReactElement, useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+import { accountApi } from 'resources/account';
+
+import { analyticsService } from 'services';
+
+import { LayoutType, RoutePath, routesConfiguration, ScopeType } from 'routes';
+import config from 'config';
+
+import MainLayout from './MainLayout';
+import PrivateScope from './PrivateScope';
+import UnauthorizedLayout from './UnauthorizedLayout';
+
+import 'resources/user/user.handlers';
+
+const layoutToComponent = {
+  [LayoutType.MAIN]: MainLayout,
+  [LayoutType.UNAUTHORIZED]: UnauthorizedLayout,
+};
+
+const scopeToComponent = {
+  [ScopeType.PUBLIC]: Fragment,
+  [ScopeType.PRIVATE]: PrivateScope,
+};
+
+interface PageConfigProps {
+  children: ReactElement;
+}
+
+const PageConfig: FC<PageConfigProps> = ({ children }) => {
+  const { route, push } = useRouter();
+
+  const { data: account, isLoading: isAccountLoading, isSuccess, isError } = accountApi.useGet();
+
+  useEffect(() => {
+    if ((!isSuccess && !isError) || !config.MIXPANEL_API_KEY) return;
+
+    analyticsService.init();
+    if(account){
+      analyticsService.setUser(account);
+    }
```

Comment:
```suggestion
    analyticsService.init();
    analyticsService.setUser(account);
```

recheck if eslint & prettier are enabled